cmd/root.go

```diff
@@ -36,7 +36,7 @@ var rootCmd = &cobra.Command{
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	if err := rootCmd.Execute(); err != nil {
+	if err:=rootCmd.Execute(); err != nil {
```

Comment:
@alenkacz shouldn't the linter have failed because of this change I made deliberately to test it out?

Reply:
maybe that's rather something that a formatter would catch? or which linter should be failing with that? https://golangci-lint.run/usage/linters/

Reply:
Ooops you're right. I got confused. Let me add an unused var and check with that.

Reply:
Yup that's working now. Thanks!